modules/custom/social_content_export/social_content_export.module

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/**
+ * @file
+ * The social_content_export module.
+ */
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\views\Views;
+
+/**
+ * Returns event location.
+ *
+ * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+ *   The entity to look for data to export.
+ *
+ * @return mixed
+ *   Return location of event to take place.
+ */
+function social_content_export_event_location(ContentEntityInterface $entity) {
+  $query = \Drupal::database()->select('node__field_event_location', 'nfel');
+  $query->addField('nfel', 'field_event_location_value');
+  $query->condition('nfel.entity_id', $entity->id());
+
+  return $query
+    ->execute()
+    ->fetchField();
+}
+
+/**
+ * Returns event host.
+ *
+ * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+ *   The entity to look for data to export.
+ *
+ * @return mixed
+ *   Returns address of event.
+ */
+function social_content_export_event_host(ContentEntityInterface $entity) {
+  $query = \Drupal::database()->select('node__field_event_address', 'ad');
+  $query->addField('ad', 'field_event_address_locality');
+  $query->condition('ad.entity_id', $entity->id());
+
+  return $query
+    ->execute()
+    ->fetchField();
+}
+
+/**
+ * Returns participant.
+ *
+ * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+ *   The entity to look for data to export.
+ *
+ * @return mixed
+ *   Return event participants.
+ */
+function social_content_export_event_participant(ContentEntityInterface $entity) {
+  $query = \Drupal::database()->select('event_enrollment_field_data', 'efd');
+  $query->addField('efd', 'name');
+  $query->condition('efd.user_id', $entity->id());
+
+  return $query
+    ->execute()
+    ->fetchField();
+}
+
+/**
+ * Returns event title.
+ *
+ * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+ *   The entity to look for data to export.
+ *
+ * @return mixed
+ *   Return event title.
+ */
+function social_content_export_event_title(ContentEntityInterface $entity) {
+  $query = \Drupal::database()->select('node_field_data', 'nfd');
+  $query->addField('nfd', 'title');
+  $query->condition('nfd.nid', $entity->id());
+
+  return $query
+    ->execute()
+    ->fetchField();
+}
+
+/**
+ * Returns event organizer.
+ *
+ * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+ *   The entity to look for data to export.
+ *
+ * @return mixed
+ *   Return event organizer.
+ */
+function social_content_export_event_organizer(ContentEntityInterface $entity) {
+  $query = \Drupal::database()->select('event_enrollment_field_data', 'eo');
```

Comment:
A database query is not needed for this. This is just a value on the node. Please look up the proper way on how to access a value of a field on an entity. Ask me for help if you get stuck.

Make sure you check if the entity has this field and provide a fallback value if it doesn't. Not all node types that we export have this field.

Hint: You're dealing with an entity reference relationship which sometimes is a bit more difficult. You'll have to fetch a user object and return its username.